streamz/core.py

```diff
@@ -708,6 +713,9 @@ def cb(self):
 
 
 class zip(Stream):
+    _graphviz_orientation = 270
```

Comment:
Does this mean that things will always be left to right?

Reply:
Left to right is controlled by the keyword `rankdir='LR'` rather than `rankdir='TD'`

This affects the orientation of the triangle shape in zip to be pointing to the right.

Reply:
Right, I assume that the triangle is pointing to the right because the pointy part of the triangle is where the stream comes "out". But that preferences one graph orientation over others.